data/dev/debugging/judge_prompts/debug_common_instructions.md

```
@@ -0,0 +1,29 @@
+  - What type of bug it is
+  - What the correct fix should be
+  - Expected vs actual values
+- **answer**: The student's submitted answer that you are grading
+- **inventory**: Valid element IDs and circuit structure from the artifact
+
+### Your Task
+
+Compare the student's **answer** field against the ground truth in the **refs** field. Use the criteria and Answer Key below to determine how well the student identified the bug and proposed a fix.
+
+### Output Format
+- Output JSON only: `{"scores": {"<criterion_id>": <float 0..1>, ...}, "overall": <0..1>}`.
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)